packages/db/src/models/granules-executions.ts

```diff
@@ -35,7 +35,8 @@ export default class GranulesExecutionsPgModel {
     return await knexTransaction(this.tableName)
```

Comment:
Is it worth specifying the return type here as well? That'd be different from the other methods in this model but maybe it's worth making slow progress towards typing everything we can?

Reply:
Just want to be consistent with upsert methods in other models.